src/main/resources/maps/MenuLevel/14x17_MenuLevel.crf

```diff
@@ -20,7 +20,7 @@ WALL WALL WALL WALL FLOOR FLOOR WALL WALL WALL WALL # # # #
 5 1 4 Key 1/0/0/0
 9.5 .5 5.5 Door 1/0/0/0
 4.5 0 13.5 PlayerTrigger 1 .5 StartGame
-3 .5 4 Pitfall 1
+3 0 4 LandMine
```

Comment:
Now the pitfall is no longer covered by the tests, so maybe just add a landmine, instead of replacing the pitfall.